dmoj/executors/compiled_executor.py

```diff
@@ -104,6 +106,20 @@ def get_compile_env(self) -> Optional[Dict[str, str]]:
     def get_compile_popen_kwargs(self) -> Dict[str, Any]:
         return {}
 
+    def _add_compiler_syscalls(self, sec: CompilerIsolateTracer) -> CompilerIsolateTracer:
```

Comment:
This is a copy of `_add_syscalls`. Let's just make that take the list of syscalls to add, and use it here.

Reply:
I completely misunderstood, let me try again.

Reply:
Alright, should be good now.